roles/edge-image-builder/templates/config-eib.yaml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1.0
+image:
+  imageType: raw
+  baseImage: {{ eib_source_image_name }}
+  outputImageName: {{ eib_output_image_name }}
+operatingSystem:
+  kernelArgs:
+  - ignition.platform.id=openstack
+  users:
+  - username: root
+    password: temporal
```

Comment:
This won't work because the password is expected to be encrypted. https://github.com/suse-edge/edge-image-builder/issues/67

Reply:
ok changed. I will test this again